manual/chapter2/history.md

```diff
@@ -0,0 +1,24 @@
+# History Setting
+At present, history has two modes: standard mode(for modern browsers) and compatible mode (for IE and old Edge).You can set history mode by `editor.config.compatibleMode` option.You aslo can set history max size by `editor.config.historyMaxSize` option.
```

Comment:
There are two ways to set History：standard mode(for modern browsers) and compatible mode (for IE and old Edge). You can set mode by `editor.config.compatibleMode`. And about the max size of History, you can set it by `editor.config.historyMaxSize`. The following shows how to use them.